addons/web_editor/static/src/js/editor/rte.js

```diff
@@ -507,6 +507,15 @@ var RTEWidget = Widget.extend({
      * @param {jQuery} $editable
      */
     _enableEditableArea: function ($editable) {
+        if ($editable.data('oe-type') === "datetime" || $editable.data('oe-type') === "date") {
+            debugger;
+            var $linkedFieldNodes = $('#wrapwrap [data-oe-id="' + $editable.data('oe-id') + '"][data-oe-field="' + $editable.data('oe-field') + '"][data-oe-model="' + $editable.data('oe-model') + '"]').not($editable);
```

Comment:
Not sure but should not all of this be moved in website instead of web_editor? Or at least not mention wrapwrap in web_editor?

Reply:
So search on whole DOM? `$('[data..`
There is no way to get `editable_area` I think here